src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1858,13 +1946,21 @@ private void OnRigidityChanged()
     private void EnqueueAction(MicrobeEditorAction action)
     {
         // A sanity check to not let an action proceed if we don't have enough mutation points
+        // Or if they are in the process of moving an organelle
         if (MutationPoints < action.Cost)
         {
             // Flash the MP bar and play sound
             gui.OnInsufficientMp();
             return;
         }
 
+        if (MovingOrganelle != null && action.Name != nameof(MoveOrganelle))
```

Comment:
I suppose this is fine as action building doesn't impact the editor state before the created action is executed.